tape/src/main/java/com/squareup/tape/FileObjectQueue.java

```diff
@@ -75,6 +85,18 @@ public FileObjectQueue(File file, Converter<T> converter) throws IOException {
       queueFile.forEach(new QueueFile.ElementVisitor() {
         int count;
         @Override public boolean read(InputStream in, int length) throws IOException {
+          final int fileLength = queueFile.fileLength - QueueFile.HEADER_LENGTH;
```

Comment:
Code style: don't make local vars final.

Name: fileContentLength ?

Reply:
Where is this in our code style? I don't like that, but I can cede.

Reply:
Done.